src/components/pages/Sources/SourceDetails/SourceSettings/General/NameNUrl.tsx

```diff
@@ -18,6 +18,11 @@ import {Permissions, usePermission} from '@permissions/base';
 
 import {MainContext} from '@contexts';
 
+type NameNUrlFormValues = {
```

Comment:
any reason this is called NameNUrl and not NameUrl?

Reply:
It was named NameNUrl at some point, don't see any problems here. On this form we can change name and url :)